sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobAsyncClient.java

```diff
@@ -313,7 +313,7 @@
      * @param tier {@link AccessTier} for the destination blob.
      * @param requestConditions {@link BlobRequestConditions}
      * @return An empty response
-     * @throws IllegalArgumentException If {@code blockSize} is less than 0 or greater than 100MB
+     * @throws IllegalArgumentException If {@code blockSize} is less than 0 or greater than 4000MB
```

Comment:
Can we make that 4GB

Reply:
Problem is. Jumbo block is up to exactly 4000MB. 4GB is more than that.